src/test/services/settings-mock.ts

```diff
@@ -60,6 +62,8 @@ export const DEFAULT_ADMIN_SETTINGS_MOCK: AdminSettings = {
 
 @Injectable()
 export class SettingsMockService {
+  private _quotas = QuotasMock;
```

Comment:
can we define it's own file for `QuotasMockService` instead of adding using existing SettingsMockService?

Reply:
For that there needs to be a separate quotas service, but we are using settings service for all settings related data.

Reply:
changes retained